clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -48,9 +48,4 @@ export const INSTALL_ZOWE = "Install Zowe Explorer";
 export const DACO_DIALECT = "DaCo";
 export const IDMS_DIALECT = "IDMS";
 export const DEFAULT_DIALECT = "COBOL";
-
-export enum PUNCH_CARD {
-    SEQUENCE_AREA_END_POS = 6,
-    INDICATOR_AREA = 7,
-    AREA_A_END_POS = 11
-}
+export const SETTINGS_TAB_CONFIG: string = "cobol-lsp.smart-tab.enabled";
```

Comment:
```suggestion
export const SETTINGS_TAB_CONFIG: string = "cobol-lsp.smart-tab";
```

Reply:
Please also rename it in package.json add the configuration schema (array and object).

Reply:
Specifying array works well, except that the values are taken as offsets (starting with 0) rather than index (starting with 1). So specifying `[5, 25, 45]` leads to tab stops at `[6, 26, 46]`.

Reply:
Strange behaviour with configuration object like this:
```json
{
		"default": [1, 15, 30, 45, 60],
		"anchors": {
			"PROCEDURE +DIVISION": [5, 25, 45],
			"DATA +DIVISION": [13, 43, 73]
}
```
Please test it on a simple program with procedure division and data division - start on a new line and keep hitting Tab key.

https://user-images.githubusercontent.com/19686979/182376742-48b52a31-a6a6-4c09-819d-d65d177b784a.mov